### DISCUSSION TITLE
Pull requests and the Issues

### DISCUSSION THREAD

**Discussion by abhishektripathi66** [eclipse/jkube#3534] [General]
Merge the valid Pull requests and close the Issues

**Comment by rohanKanojia** [comment]
@abhishektripathi66 : Could you please elaborate? Is there any pull request you would like us to take look into?

**Reply by abhishektripathi66** [reply]
I Mean there are many good first issues which have the pull requests linked, if those could be addressed it would be better, which will help us to address those issues better.

**Reply by abhishektripathi66** [reply]
There are many good first issues which has the pull request linked but there isn't any assignee.

**Reply by rohanKanojia** [reply]
@abhishektripathi66 : Thanks, will try to clean up pull requests and issues in coming weeks.
